### DISCUSSION TITLE
Outside Collaborators Org-Wide v1.0.0

### DISCUSSION THREAD

**Discussion by pattacini** [mesh-iit/outside-collaborators#52] [Show and tell]
This is the first official release of the automation!
Refer to the main [README][1] for the documentation.

[1]: https://github.com/icub-tech-iit/outside-collaborators

<hr /><em>This discussion was created from the release <a href='https://github.com/icub-tech-iit/outside-collaborators/releases/tag/v1.0.0'>Outside Collaborators Org-Wide v1.0.0</a>.</em>

**Comment by JeroenKnoops** [comment]
I can't stress enough how useful this tool is.
We're using this at massive scale!

Congrats with the release! 🚀

**Reply by pattacini** [reply]
Thanks heaps @JeroenKnoops for your nice and supportive words!
Really appreciated!

I'm also very happy that the tool turns to be useful for your org!
